buildcheck/views/validationView_reviewer.py

```diff
@@ -0,0 +1,19 @@
+"I'm going to edit this template later"
```

Comment:
No need to leave this bare string here.

In the future you can leave comments `# like this`